components/templates/Cargo.toml

```diff
@@ -12,6 +13,11 @@ toml = "0.4"
 csv = "1"
 serde_json = "1.0"
 error-chain = "0.12"
+crypto-hash = "0.3.1"
+chrono = "0.4"
+reqwest = "0.9"
+url = "1.5"
+clap = "2"
```

Comment:
I don't think clap is needed?

Reply:
The reason I loaded `clap` into this package was for the `crate_version` macro for use in the header. Seeing as it's already in the binary, the additional overhead should be minimal.